### PR TITLE
Example Implementation

### PR DESCRIPTION
**Pull request by dearshrewdwit**
Completed:
- Standard Criteria
- Extension 1

### CODE REVIEW

File: src/components/Checkboxes.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  return (
+    <ul>
+      <li>
+        <label>
+          <input
+            name="spend-time"
+            type="checkbox"
+            value="swimming"
+            onChange={handleChange}
+            checked={timeSpent.includes("swimming")}
```
Comment (dearshrewdwit) [inline src/components/Checkboxes.jsx:12]: An example comment!

File: src/components/SurveyForm.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import {
```
Comment (Njoyp) [inline src/components/SurveyForm.jsx:2]: Could you explain something about inheritance again (like how parent and child work together)?

File: src/components/index.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import RadioButtons from './RadioButtons'
+import Checkboxes from './Checkboxes'
+import AnswersList from './AnswersList'
+import AnswersItem from './AnswersItem'
+import SurveyForm from './SurveyForm'
+
```
Comment (Njoyp) [inline src/components/index.jsx:6]: What does this file do? Does it store the components and why?

File: src/components/Main.jsx [hunk 4]
```diff
@@ -1,15 +1,65 @@
...
+  const [formData, setFormData] = useState(initialFormData)
+  const [answers, setAnswers] = useState([])
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    console.log(formData)
+    setAnswers([...answers, formData])
+    setFormData({
+      color: '',
+      timeSpent: [],
```
Comment (Max7oo) [inline src/components/Main.jsx:26]: Figuring out that the timeSpent had to go in an array.

File: src/components/Main.jsx [hunk 5]
```diff
@@ -1,15 +1,65 @@
...
+      review: '',
+      username: '',
+      email: ''
+    })
+  }
+
+  const handleChange = (e) => {
+    const {name, type, value, checked} = e.target
+    if (type === "checkbox" && name === "spend-time") {
```
Comment (AidanvG) [inline src/components/Main.jsx:35]: Please could you run through this whole if statement again.

File: src/components/Main.jsx [hunk 6]
```diff
@@ -1,15 +1,65 @@
...
+      color: '',
+      timeSpent: [],
+      review: '',
+      username: '',
+      email: ''
+    })
+  }
+
+  const handleChange = (e) => {
+    const {name, type, value, checked} = e.target
```
Comment (Max7oo) [inline src/components/Main.jsx:34]: How to keep timeSpent after refreshing.

**Pull request closed without merging**